### PR TITLE
Clean up and better cross-link new docsv2 documents

### PR DESCRIPTION
**Pull request by joelhans**
##### Summary

Cleaning up some of the `docsv2` project now that the whole package is deployed. I also added code to enable better icons in dark mode on the [Get Netdata page](https://learn.netdata.cloud/docs/get). (cc @OdysLam).

##### Component Name

area/docs

### CODE REVIEW

File: docs/collect/application-metrics.md [hunk 1]
```diff
@@ -69,6 +69,10 @@ list](/collectors/COLLECTORS.md#service-and-application-collectors) that can run
 
 ## What's next?
 
+If you haven't yet see the [supported collectors list](/collectors/COLLECTORS.md) give it a once-over for any additional
```
Comment (aabatangle) [inline docs/collect/application-metrics.md:72]: see --> seen

File: docs/monitor/configure-alarms.md [hunk 2]
```diff
@@ -60,10 +60,8 @@ Save the file and [reload Netdata's health configuration](#reload-health-configu
 
 ### Silence an individual alarm
 
-Many Netdata users don't need all the default alarms enabled. Instead of disabling any given alarm, or even _all_
-alarms, you can silence individual alarms by changing one line in a given health entity. 
-
-To silence any single alarm, change the `to:` line to `silent`.
+Instead of disabling an alarm altogether, or even disabling _all_ alarms, can silence individual alarms by changing one
```
Comment (aabatangle) [inline docs/monitor/configure-alarms.md:63]: missing 'you' before 'can silence'?

File: docs/quickstart/infrastructure.md [hunk 3]
```diff
@@ -9,17 +9,22 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/docs/quickstart/
...
 Together, the Netdata Agent and Netdata Cloud create a powerful, infinitely-scalable infrastructure monitoring solution.
 
-The Netdata Agent uses zero-configuration collectors to gather metrics from every application and container instantly.
-The distributed data model lets you monitor everything without a slow and troublesome centralized data lake for your
-infrastructure's metrics, reducing the resources you need to invest in metrics retention. Netdata Cloud unifies all the
-metrics from these distributed nodes on customizable, interactive, and real-time visualizations.
+The Netdata Agent uses zero-configuration collectors to gather metrics from every application and container instantly,
+and uses Netdata's [distributed data architecture](/docs/store/distributed-data-architecture.md) to store metrics
+locally. Without a slow and troublesome centralized data lake for your infrastructure's metrics, you reduce the
+resources you need to invest, and the complexity of, monitoring your infrastructure.
```
Comment (aabatangle) [inline docs/quickstart/infrastructure.md:15]: invest --> 'invest in' since you're using an preposition for 'complexity of'

**Review by aabatangle — DISMISSED** [review]

**Review by OdysLam — APPROVED** [review]

File: docs/quickstart/infrastructure.md (hunk 3 above)
Comment (joelhans) [reply docs/quickstart/infrastructure.md]: Thanks much, @aabatangle! Fixed these.

**Review by aabatangle — APPROVED** [review]

**Review by jlbriston — APPROVED** [review]

**Pull request merged**